apps/load-tests/tests/src/processors/models-tree-stateless.ts

```diff
@@ -103,7 +103,7 @@ function createModelsTreeProvider(context: ScenarioContext, events: EventEmitter
       }
       try {
         const schemaInfo = await Schema.startLoadingFromJson(schemaJson, schemaContext);
-        if (schemaInfo !== undefined && schemaInfo.schemaKey.matches(schemaKey, matchType)) {
+        if (schemaInfo !== undefined && schemaInfo.schemaKey.matches(schemaKey as SchemaKey, matchType)) {
```

Comment:
Can we change function definition from `schemaKey: Readonly<SchemaKey>` to `schemaKey: SchemaKey`?

Reply:
`ecschema-metadata@4.x` defines `ISchemaLocater` with readonly schema keys, changing them to non-readonly will make our regression tests fail to build. This is test code, so I think it's fine to use `as` as a workaround.